src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser.g4

```diff
@@ -878,7 +878,8 @@ javadocTag: AUTHOR_LITERAL (WS | NEWLINE)* description?
 
       | RETURN_LITERAL (WS | NEWLINE)* description?
 
-      | SEE_LITERAL (WS | NEWLINE)* reference? (STRING | htmlElement)* (WS | NEWLINE)* description?
+      | SEE_LITERAL (WS | NEWLINE)* reference? ((WS | NEWLINE)+ STRING | htmlElement)*
+      (WS | NEWLINE)* ((WS | NEWLINE)+ description)?
```

Comment:
@romani 
`@see something` won't match in this case: ` something` matches `reference?` and then goes `NEWLINE` which doesn't match neither `(WS | NEWLINE)+ STRING | htmlElement)` because after `(WS | NEWLINE)+` must be `STRING | htmlElement` nor `((WS | NEWLINE)+ description)` because after `(WS | NEWLINE)+` must be `description`